src/components/views/elements/PersistedElement.tsx

```diff
@@ -177,8 +182,9 @@ export default class PersistedElement extends React.Component<IProps> {
         Object.assign(child.style, {
             zIndex: isNullOrUndefined(this.props.zIndex) ? 9 : this.props.zIndex,
             position: 'absolute',
-            top: parentRect.top + 'px',
-            left: parentRect.left + 'px',
+            top: '0',
+            left: '0',
```

Comment:
Do we need to assign 0s here?

Reply:
Yep, otherwise the element gets placed somewhere offscreen

Reply:
Since it's always the same could it be set in CSS? Am I missing something?